changes/ce/fix-12814.en.md

```diff
@@ -0,0 +1,4 @@
+Handle several errors in `/clients/{clientid}/mqueue_messages` and `/clients/{clientid}/inflight_messages` APIs:
+
+- Internal timeout, which means that EMQX failed to get the list of Inflight/Mqueue messages within the default timeout of 5 s. This error may occur when the system is under a heavy load. The API will return 500 `{"code":"INTERNAL_ERROR","message":"timeout"}` response and log additional details.
```

Comment:
Could be that the client process is stuck. Maybe even worth logging the current stacktrace from process_info

Reply:
Added a client proc stacktrace to the log event.